spec/system/task_spec.rb

```diff
@@ -1,24 +1,23 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { FactoryBot.create(:project) }
```

Comment:
設定ファイルに追記を行うことで省略が可能です。
コードの全体量が減って見通しが良くなるので、積極的に省略していきましょう。
```suggestion
  let(:project) { create(:project) }
```

Reply:
コミットID[201a6d3](https://github.com/SakitaDaiki0623/rspec_app_exam/pull/1/commits/201a6d327c87575c6231e18e571de288d688d63e)で修正しました。